src/components/HeatMap/Row/Row.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Cell from '../Cell';
+import { Wrapper, RowName, Text } from './Row.styles';
+
+
+export default function Row({
+  data, dayLabel, row, getActiveCell, activeCell,
```

Comment:
If `row` is actually the number representing the weekday maybe call it `weekday` so it's more descriptive